stan/math/prim/mat/functor/map_rect_concurrent.hpp

```diff
@@ -102,8 +102,7 @@ map_rect_concurrent(
 
 #ifdef STAN_THREADS
   if (num_threads > 1) {
-    const int num_big_threads
-        = (num_jobs - num_jobs_per_thread) % (num_threads - 1);
+    const int num_big_threads = num_jobs % num_threads;
```

Comment:
What are big threads?

Reply:
In case the number of jobs and threads do not line up and we have more jobs than threads there will be a few threads with one extra job. That are the big threads.

Reply:
Gotcha! Makes sense now.